DaFont/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+        "name": "Jake.#1111",
```

Comment:
Please remove your user tag (#*1111*) from here.

Reply:
> I never got that before  . Is this new?

Reply:
Nope always been there, if it is in your others, please change them. We might've missed it.

Reply:
Yeah you did, will do. For now,  I've fixed this one

Reply:
If you could also fix the others and increment their versions, you can commit them to your fork and this PR will merge all of them. There won't be need for another PR. It's up to you.